app.js

```diff
@@ -1,9 +1,19 @@
 const express = require("express");
-const { getCategories, getEndpoints } = require("./controllers/controller");
+const {
+  getCategories,
+  getEndpoints,
+  getReviewId,
+} = require("./controllers/controller");
 const app = express();
 
 app.get("/api/categories", getCategories);
 
 app.get("/api", getEndpoints);
 
+app.get("/api/review/:review_id", getReviewId);
+
+app.use((err, req, res, next) => {
+  res.status(404).send("review id does not exist");
```

Comment:
So this error handling middleware at the moment is only set to always return a 404 and 'review id does not exist' here.

We don't want to be sending specific error messages like this here, we want to keep it generic to whatever is passed into it. The Promise.reject in your model actually deals with setting up the error and then passing it to this middleware. Have another check of the notes on `Error Handling` and refactor this... essentially what you want is:

```
  if (err.status && err.msg) {
    res.status(err.status).send({ msg: err.msg });
  }
```
  
  Where err.status and err.msg is whatever has been passed to it from the promise rejection in the model.